src/team_lib/ucx/xccl_ucx_lib.c

```diff
@@ -99,6 +99,12 @@ static ucs_config_field_t xccl_tl_ucx_context_config_table[] = {
      UCS_CONFIG_TYPE_UINT
     },
 
+    {"PRE_MEM_MAP", "0",
+     "register communication buffer with UCX duing collective init to populate registration cache",
```

Comment:
so, if "1" then src/dst buffers are registered during coll_init. And if "2" - all user mallocs are registered?

Reply:
yes